the-grid.html

```diff
@@ -149,6 +149,17 @@
                         value: false,
                         reflectToAttribute: true,
                         observer: '_attachEvents'
+                    },
+
+                    /**
+                     * Enable infinite automatic extension of the grid on resize and move.
+                     * @type {boolean}
+                     */
+                    selfResize: {
```

Comment:
otherwise, the changes are done, I'll finalize the new PR when we get this question out of the way